packages/ia-zendesk-help-widget/src/ia-zendesk-help-widget.js

```diff
@@ -0,0 +1,148 @@
+import { html, css, LitElement } from 'lit-element';
+import { LazyLoaderService } from '@internetarchive/lazy-loader-service';
+import loaderIcon from './assets/icon-loader';
+import questionIcon from './assets/icon-question';
+
+class ZenDeskHelpWidget extends LitElement {
+  static get properties() {
+    return {
+      widgetSrc: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.widgetSrc = '';
+    this.buttonVisible = true;
+    this.isLoading = false;
```

Comment:
I'm not sure if this was intentional, but if you want `buttonVisible` and `isLoading` to be reactive, add them to the `static get properties() {}`. Right now if you change either of them, the UI won't update.